OCPBUGS-24557: Explain spec.clusterImageSetNameRef and spec.clusters.clusterImageSetNameRef in SiteConfig CR (#71724)
openshift/openshift-docs: modules/ztp-precaching-ztp-config.adoc

            - name: "ens1f0"
              macAddress: "AA:BB:CC:11:22:33"
----

> <1> References the cluster image set that is used for deployment, unless you specify a different image set in the `spec.cluster.clusterImageSetNameRef` field.

Reviewer: nit. clusters and not cluster

> <1> References the cluster image set that is used for deployment, unless you specify a different image set in the `spec.clusters.clusterImageSetNameRef` field.

Reviewer 2: ```suggestion
<1> Specifies the cluster image set used for deployment, unless you specify a different image set in the `spec.clusters.clusterImageSetNameRef` field.
```
- Would `specifies` be more accurate than `references` here?
- Edit for minimalism

> <2> References the cluster image set that is used to deploy an individual cluster. If defined, it overrides the `spec.clusterImageSetNameRef` at site level.

Reviewer 2: ```suggestion
<2> Specifies the cluster image set used to deploy an individual cluster. If defined, it overrides the `spec.clusterImageSetNameRef` at the site level.
```
- Would `specifies` be more accurate than `references` here?
- Edit for minimalism
